### PR TITLE
JOSS actions.

### PR DESCRIPTION
**Pull request by AlexBuccheri**
See if the CI runs for me

### CODE REVIEW

**Comment by AlexBuccheri** [comment]
I note that my attempt (probably) would have worked, but the tex installation was way too slow (> 10 mins)

```yaml
name: Open Journals PDF Generator

on: [push]

jobs:
  paper:
    runs-on: ubuntu-latest
    name: Paper Draft
    steps:
      - name: Checkout
      - uses: actions/checkout@v3
      - name: Install Dependencies
        run: |
          sudo apt-get update && sudo apt-get install -y pandoc pandoc-citeproc texlive texlive-latex-extra texlive-science texlive-publishers
      - name: Compile PDF
        run: |
          pandoc --filter pandoc-citeproc --bibliography=paper/paper.bib -s paper/paper.md -o paper/paper.pdf --template paper/latex.template
      - name: Upload PDF
      - uses: actions/upload-artifact@v3
        with:
          name: joss-paper
          path: paper/paper.pdf
```

**Comment by AlexBuccheri** [comment]
@speckhard The problem appeared to be this line:
`uses: openjournals/openjournals-draft-action@v.1.0`

Works fine after removing it. I also fixed the paths as our data is in` $ROOT/paper`

Unfortunately unlike Gitlab, I can't find a linter for Github actions (looks like one can download user-developed, command-line linters), so was just trial+error pushing until I found the problem

File: paper/paper.md [hunk 1]
```diff
@@ -53,32 +53,38 @@ well-suited for the generation of benchmark-quality data, and to serve as a refe
 especially those relying on pseudopotentials.
```
Comment (AlexBuccheri) [inline paper/paper.md:53]: Trivial reformatting of the file, for max line length. Also fixed a word duplicate

**Comment by AlexBuccheri** [comment]
Artefact available here: https://github.com/exciting/excitingtools/actions/runs/3841170753

**Review by speckhard — APPROVED** [review]
LGTM, thank you!

File: paper/paper.md (hunk 1 above)
Comment (speckhard) [reply paper/paper.md:53]: good find, thank you!

**Pull request merged**